### DISCUSSION TITLE
Text wrap in column cells

### DISCUSSION THREAD

**Discussion by wlclifford** [pawelsalawa/sqlitestudio#5171] [Q&A]
I just updated from 3.4.4 to 3.4.10 and now the cells in columns wrap where previously there was an ellipsis to signify text view is truncated.  I am often working with medical terminology notorious for long names and the wrap is distracting when viewing table data or query results.  Is there any way to turn this off just as it can be done in the SQL editor?  Thanks,

**Comment by pawelsalawa** [comment]
I'm not quite sure what you mean. For what I understood you mean values like these:
![image](https://github.com/user-attachments/assets/9a1d27cd-914c-4603-8884-a579d7dcffe9)

I see them with ellipsis:
![image](https://github.com/user-attachments/assets/1851797e-9ad7-4fd1-a351-aec4d028544e)

Don't you?

**Reply by wlclifford** [reply]
I see it like this:
![image](https://github.com/user-attachments/assets/709577cc-1b39-48df-b1f5-ff700765c261)

**Reply by pawelsalawa** [reply]
I'm unable to reproduce it. Could you share one value of such problematic cell here? Or maybe over the email support@sqlitestudio.pl?

Secondly, what is your operating system?

**Reply by wlclifford** [reply]
Thanks for being so responsive.  I had tried closing and reopening sqlitestudio a few times to no avail in stopping the wrap so assumed this was a new feature after the update today.  However, knowing now that this was not a new feature, I rebooted Windows 11 and I'm mercifully back to ellipses.  I had updated to the latest version of Win 11 about 2 weeks ago but there were at least 2 reboots between that and updating sqlitestudio.  So now I can't reproduce the wrapping.  It may never happen again to me or anyone else but if it does, it might point to something in releases after 3.4.4 that doesn't agree with Win 11.  I'll provide an update if there is a recurrence.

Love the product.  Thanks again.

BTW - here is the raw tab separated text of the 2 cells in the top row of the screenshot -  33361000087101	COMIRNATY (original strain) messenger ribonucleic acid 10 micrograms per 0.2 milliliter suspension for injection BioNTech Manufacturing GmbH (real clinical drug).  This is clean data - i.e. no hidden characters lurking in there.

**Reply by wlclifford** [reply]
I've figured out how to reproduce this - nothing to do with Win 11 and rebooting and other mental mechanisms that I invoked to explain what is unexpected.  If you hold the control key and mouse scroll up to zoom in, the lines start to wrap in the cell.  With scrolling in and out, it is possible without closing and reopening the app to reset this to normal by getting just the right magnification but it is fiddly.   I often multiselect with CTRL and accidentally started scrolling with the mouse before letting go of the key which is how I brought this on this time.  Probably the same thing happened last time and not again until now which shows that it doesn't happen very often for me.  At least the conditions leading to it are known now!

![image](https://github.com/user-attachments/assets/34a41f1f-c64e-4d8e-8cfa-81133a8cc924)

**Reply by pawelsalawa** [reply]
In this case it appears that the problem is not actually related to upgrading 3.4.4 to 3.4.10, but it's rather accidental. I have reproduced it and it happens when you decrease font size (using ctrl+wheel) to smaller than minimum row height.

In that case it seems to make sense that you see more lines isn't it?

**Reply by wlclifford** [reply]
Yes, can see more lines but there are glimpses of the top of text in the wrap.  The normal behavior is just seeing the ellipsis.  One might expect that as you zoom in, the ellipsis would move to the left and while you see less info in cell,  you wouldn't see text fragments peeking up from the bottom which make it difficult to read when scanning a page.  This isn't a big deal for me since it doesn't happen very often and if it does, I now know the background.  The natural reaction when seeing this is that there is something corrupted in the display either by app rendering or the OS itself.  If the rendering isn't adjusted to hide the wrap fragments, would be good to mention this in the documentation.  For me, it doesn't make much difference now since I love the app and can readjust if it bugs me.  I might be more affected than most because my columns tend to have lots of text.
